.github/workflows/workflow.yml

```diff
@@ -13,10 +13,8 @@ jobs:
           - macos-latest
           - ubuntu-latest
         ocaml-version:
-          - 4.10.0
-          - 4.10.1
-          - 4.11.0
-          - 4.11.1
+          - 4.13.0
+          - 4.13.1
```

Comment:
Not so sure that deprecating support for 4.10.* and 4.11.* is what we want but I can't think of any way we can continue to support the older versions if we want to also support the newer version

Reply:
Yup that's fine the ASTs are incompatible so we have to deprecate the old versions.